examples/node-server/index.js

```diff
@@ -53,6 +53,7 @@ server.get('*', async (request, response) => {
     // initialState: { ... } // <- This would also be available
   })
 
+  response.writeHead(status || 200, statusText || 'ok', headers)
```

Comment:
I think it should be simply `statusText` not `statusText || 'ok'`. Passing `undefined` by default will supposedly generate default message for the status code (https://nodejs.org/api/http.html#http_response_statusmessage), which is better default for non-200 status codes.

Reply:
I believe I tried with undefined or null but then it didn't pickup the headers as the third parameter. I see now in an example there that you are supposed to pass the headers as the second parameter if statusText is undefined. I'll try that, thanks!